### PR TITLE
524-handle-text-overflow-in-resource-cards-for-small-screens

### PR DESCRIPTION
**Pull request by Xerxi88**

### CODE REVIEW

**Comment by Xerxi88** [comment]
After: 
image
image


Before: 
image
image

**Review by NataliaR-BCN — COMMENTED** [review]
Sorry, but I think this does not solve the overflow of long texts in the Resource Card on small screens. 
Contact me on Discord if you need any help, please.
errorResourceCard

File: front/src/components/organisms/Navbar.tsx [hunk 1]
```diff
@@ -20,7 +21,7 @@ const NavbarStyled = styled(FlexBox)`
     color: ${colors.white};
   }
 
-  @media (max-width: 468px) {
+  @media (max-width: 480px) {
```
Comment (kevinmamaqi) [inline front/src/components/organisms/Navbar.tsx:24]: Utilizar estilos de device.Mobile/Tablet, etc...

**Review by kevinmamaqi — CHANGES_REQUESTED** [review]

File: front/src/components/molecules/SelectLanguage.tsx [hunk 2]
```diff
@@ -19,12 +19,12 @@ const DropdownLang = styled.select`
...
     outline: 0 none;
   }
-  @media ${device.Tablet} {
+  @media ${device.Mobile} {
     background-color: ${colors.white};
     margin: 0;
     padding: 0.47rem;
-    color: ${colors.gray.gray3};
-  }
+       color: ${colors.gray.gray3};
```
[the remark this answers is not in the document]
Comment (Xerxi88) [reply front/src/components/molecules/SelectLanguage.tsx:26]: none, the tabulation may be reflected

File: front/src/components/organisms/Navbar.tsx [hunk 3]
```diff
@@ -21,16 +22,16 @@ const NavbarStyled = styled(FlexBox)`
     color: ${colors.white};
   }
 
-  @media (max-width: 468px) {
-    background-color: ${colors.white};
+  @media  ${device.Mobile} {
+    background-color: ${colors.black};
```
[the remark this answers is not in the document]
Comment (Xerxi88) [reply front/src/components/organisms/Navbar.tsx:26]: Yes, the correct color would be "black.black3" for example, finally I change it to "gray.gray5" to correctly follow the design.

**Review by NataliaR-BCN — APPROVED** [review]

**Review by kevinmamaqi — APPROVED** [review]

**Pull request merged**